sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -67,8 +67,6 @@
     </EmbeddedResource>
 
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" PrivateAssets="All" VersionOverride="[3.14.2, 4.0.0)" />
-    <PackageReference Include="System.Net.Http" PrivateAssets="All" />
```

Comment:
There is an open track one pull request that was brought to my attention stating that `System.Net.Http` is no longer needed and the reference will be removed.    The Active Directory code paths are not active in track two.